test/browser/main.js

```diff
@@ -1,4 +1,19 @@
 var originalConsoleWarn = console.warn;
+// Version of PhantomJS that's being used by `mocha-phantomjs` does not enabled `intl`
+// which causes most tests to fail because it relies on `Intl` to be in `Window/Global`
+// This can be removed once the library is updated to use latest version that supports it.
+var areIntlLocalesSupported = require('intl-locales-supported');
+if (global.Intl) {
+    if (!areIntlLocalesSupported(['en-US'])) {
```

Comment:
I assume this means if `areIntlLocalesSupported` tests true, we're all good and no shims are necessary